doc/release-notes.md

```diff
@@ -1,4 +1,4 @@
-Bitcoin Core version 0.11.2 is now available from:
+Bitcoin Classic version 0.11.2 is now available from:
 
   <https://bitcoin.org/bin/bitcoin-core-0.11.2/>
```

Comment:
Probably hold off on this one as the file will be replaced with new release info